### PR TITLE
JavaScript in JSX with Curly Braces

### PR DESCRIPTION
**Pull request by imnotmoon**
asdf

### CODE REVIEW

File: Describing the UI/javascript-in-jsx-with-curly-braces.md [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+      src="https://i.imgur.com/7vQD0fPs.jpg"
+      alt="Gregorio Y. Zara"
+    />
+  );
+}
+```
+
+<img width="593" alt="스크린샷 2022-08-31 오전 1 09 42" src="https://user-images.githubusercontent.com/53335940/187486972-e36f790b-523d-4695-949b-4fe34b0997f2.png">
+
+여기서 `"https://i.imgur.com/7vQD0fPs.jpg"`와 `"Gregorio Y. Zara"`가 string으로서 전달되었습니다.
```
Comment (Songycs) [inline Describing the UI/javascript-in-jsx-with-curly-braces.md]:
```suggestion
여기서 `"https://i.imgur.com/7vQD0fPs.jpg"`와 `"Gregorio Y. Zara"`는 string으로 전달되었습니다.
```

File: Describing the UI/javascript-in-jsx-with-curly-braces.md [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+  return (
+    <h1>{name}'s To Do List</h1>
+  );
+}
+
+```
+
+<img width="578" alt="스크린샷 2022-08-31 오전 1 19 52" src="https://user-images.githubusercontent.com/53335940/187488887-d40bc4c5-3828-466d-ba02-e71294d2f476.png">
+
+`name` 변수를 'Gregorio Y. Zara'에서 'Hedy Lamarr'로 바꿔보세요. 바뀐 점을 눈치 채셨나요?
```
Comment (Songycs) [inline Describing the UI/javascript-in-jsx-with-curly-braces.md]:
```suggestion
`name` 변수를 `Gregorio Y. Zara`에서 `Hedy Lamarr`로 바꿔보세요. 바뀐 점을 눈치 채셨나요?
```

File: Describing the UI/javascript-in-jsx-with-curly-braces.md [hunk 3]
```diff
@@ -0,0 +1,494 @@
...
+
+오직 두 가지 방법으로만 중괄호를 JSX와 함께 사용할 수 있습니다.
+
+1. **텍스트로서** JSX 태그 안에 바로 넣기 : `<h1>{name}'s To Do List</h1>`는 동작 합니다. 하지만 `{tag}>Gregorio Y. Zara's To Do List</{tag}>`는 동작하지 않습니다.
+2. **속성으로서** `=`기호 바로 다음에 사용하기 : `src={avatar}`는 `avatar`의 값을 읽습니다. 하지만 `src="{avatar}"`는 `{avatar}`라는 string 값을 전달할 것입니다.
+
+## "이중 중괄호" 사용하기: CSS와 다른 객체
+
+string, number, 그리고 다른 JavaScript 표현식들 뿐만 아니라, 객체를 JSX에 전달하는 것도 가능합니다. 객체들은 `{ name: "Hedy Lamarr", inventions: 5 }`와 같이 원래부터 중괄호를 이용해 표현됩니다. 그래서 JS 객체를 JSX에 전달하기 위해서는 또 다른 중괄호 쌍으로 객체를 감싸주어야 합니다. `person={{ name: "Hedy Lamarr", inventions: 5 }}` 이렇게요.
+
```
Comment (Songycs) [inline Describing the UI/javascript-in-jsx-with-curly-braces.md:107]: You may see this with inline CSS styles in JSX. React does not require you to use inline styles (CSS classes work great for most cases). But when you need an inline style, you pass an object to the style attribute:

문장이 빠진 것 같습니다~

File: Describing the UI/javascript-in-jsx-with-curly-braces.md [hunk 4]
```diff
@@ -0,0 +1,494 @@
...
+    color: 'pink'
+  }
+}>
+```
+
+이제부터 여러분이 JSX 내부에서 `{{`와 `}}`를 본다면, 그냥 JSX 중괄호 안에 있는 단순한 객체라는걸 알아차릴 수 있게 될겁니다.
+
+> **주의 사항**
+>
+> 인라인 `style` 속성은 카멜케이스(camelCase)로 작성됩니다. 예를 들어 HTML 코드인 `<ul style="background-color: black">`는 컴포넌트에서 `<ul style={{ backgroundColor: 'black' }}>` 이렇게 작성되어야 합니다.
```
Comment (Songycs) [inline Describing the UI/javascript-in-jsx-with-curly-braces.md]:
```suggestion
> 인라인 `style` 속성은 카멜케이스(camelCase)로 작성됩니다. 예를 들어 HTML 코드인 `<ul style="background-color: black">`는 컴포넌트에서 `<ul style={{ backgroundColor: 'black' }}>` 와 같이 작성되어야 합니다.
```

**Comment by Songycs** [comment]
([CodeSandbox](https://codesandbox.io/s/lg4tki?file=/App.js&from-sandpack=true)에서 직접 코드를 작성할 수 있습니다.)
([CodeSandbox](https://codesandbox.io/s/zzszdd?file=%2FApp.js&from-sandpack=true)에서 정답을 확인할 수 있습니다.)

**Comment by imnotmoon** [comment]
Updates
[078e795](https://github.com/LINE-NEXT-LEVEL/React-docs-beta-KOR/pull/10/commits/078e79577b97f79aff1650848f40d8db7d4d76ce) [31df5e8](https://github.com/LINE-NEXT-LEVEL/React-docs-beta-KOR/pull/10/commits/31df5e8dee1f9c62502125f06a7734d86a04f179) CodeSandbox 관련 텍스트 수정 ('CodeSandbox에서 확인할 수 있습니다'로 통일)

[feea480](https://github.com/LINE-NEXT-LEVEL/React-docs-beta-KOR/pull/10/commits/feea4804830e0d79b321e48e0ef805a3957a7523) 누락된 번역 추가

**Review by hyehyeonmoon — APPROVED** [review]
고생하셨습니다!

**Review by Songycs — APPROVED** [review]
👍 👍 👍

**Pull request merged**